testfiles/positive/sw-versions

```diff
@@ -1 +0,0 @@
-OMNECT-gateway-devel 4.0.17.123456
```

Comment:
where does sysinfo read the software version from? do the tests still run and can I still start ods on my host?

Reply:
sysinfo will use the Name and the Version from /etc/os-release. Tests still works, because in this case the feature "mock" will use and i added hardcoded the needed values here: https://github.com/omnect/omnect-device-service/pull/121#discussion_r1867184418

Reply:
and yes you can still use it on your host system